prime-router/settings/staging/0170-ucsd-onboarding-receiver-settings.yml

```diff
@@ -0,0 +1,109 @@
+# To submit this to staging, run:
+# ./prime multiple-settings set --env staging --input ./settings/staging/0170-ucsd-onboarding-receiver-settings.yml
+#
+# To submit this in localhost, run:
+# ./prime multiple-settings set --input ./settings/staging/0170-ucsd-onboarding-receiver-settings.yml
+#
+# Description:
+#   These following settings are for UCSD
+#   Result Receiver (ucsd.etor-nbs-results)
+#
+---
+- name: "ucsd"
+  description: "University of California San Diego"
+  jurisdiction: "STATE"
+  stateCode: CA
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders: []
+  receivers:
+    - name: "etor-nbs-results"
+      organizationName: "ucsd"
+      topic: "etor-ti"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: false
+        receivingApplicationName: "EPIC-INNERCONNECT"
+        receivingApplicationOID: null
+        receivingFacilityName: "CA"
+        receivingFacilityOID: null
```

Comment:
Can we remove all the properties that are using the default value? It will make it easier to maintain and to know what we're actually changing